while 1:
	try:
		x = int(input('pls enter a number:'))
		break
		#except (RuntimeError, TypeError, NameError)
	except ValueError:
		print('Oops!try again')
		
		
		

